src/tests/ftest/harness/basic.py

```diff
@@ -46,7 +46,7 @@ def test_load_mpi(self):
             self.fail("Orterun initialization failed: {}".format(error))
 
         try:
-            Mpirun(None, mpitype="mpich")
+            Mpirun(None, mpi_type="mpich")
```

Comment:
This seems odd, is it related or required?

Reply:
It's a bug-fix: https://github.com/daos-stack/daos/blob/457d565b632be2b663f7f1316cb2f95051dbb77a/src/tests/ftest/util/job_manager_utils.py#L373-L376